src/Stratis.Bitcoin.IntegrationTests/CoinViewTester.cs

```diff
@@ -75,8 +75,7 @@ public uint256 NewBlock()
 
         public uint256 Rewind()
         {
-            this.hash = this.coinView.Rewind();
-            this.blockHeight--;
+            this.hash = this.coinView.Rewind(--this.blockHeight);
```

Comment:
Is this correct? This is different functionality to previously. This will pass `blockHeight-1` to `Rewind`

Reply:
Previously we implicitly passed `blockHeight - 1` to `Rewind`. Now its explicit.